### PR TITLE
BG merge <area> to skip conflict with doube В В

### PR DESCRIPTION
**Pull request by BossNeo**
and adding native test sentence to turn on light

### CODE REVIEW

**Comment by hristo-atanasov** [comment]
Looks good on first glance. I will take a closer look trogh your files tomorrow. I would also involve and ask for his oppinion @dobromir-hristov.

File: sentences/bg/climate_HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -3,11 +3,9 @@ intents:
...
       - sentences:
-          - "[каква | колко] е <temp>"
-          - "[каква | колко] е <temp> [в | на] [ <area>]"
-          - "[каква | колко] <temp> е"
-          - "[каква | колко] <temp> е [в | на] [ <area>]"
-          - "колко [е] (топло | горещо | студено | хладно) [е] [(в | на) <area>]"
-          - "(топло | горещо | студено | хладно) ли е [(в | на) <area>]"
+          - "<what_is>  <temp>[ <area>]"
+          - "<what_is> <temp> е  [ <area>]"
+          - "колко [е] (топло | горещо | студено | хладно) [е] <area>]"
```
Comment (dobromir-hristov) [inline sentences/bg/climate_HassClimateGetTemperature.yaml:8]:
```suggestion
          - "<what_is> (топло | горещо | студено | хладно) [е] <area>]"
```

**Review by dobromir-hristov — APPROVED** [review]
LGTM one small extra change i saw. We can clean this up further in a next PR if needed.

**Review by hristo-atanasov — COMMENTED** [review]
OK

**Pull request merged**